setup/dashboard/main.tf

```
@@ -0,0 +1,37 @@
+resource "google_cloud_run_service" "dashboard" {
+  name     = "fourkeys-grafana-dashboard"
+  location = var.google_region
+
+  template {
+    spec {
+      containers {
+        ports {
+          container_port = 3000
+        }
+        image = "gcr.io/${var.google_project_id}/fourkeys-grafana-dashboard"
+        env {
+          name  = "PROJECT_NAME"
+          value = var.google_project_id
+        }
+      }
+      service_account_name = var.fourkeys_service_account_email
+    }
+  }
+
+  traffic {
+    percent         = 100
+    latest_revision = true
+  }
+
```

Per #216, let's add labels to the resources we create, so they can be bulk-deleted when troubleshooting

```suggestion
  metadata {
    labels = {"created_by":"fourkeys"}
  }
```